tools/upgrade.sh

```diff
@@ -22,6 +22,7 @@ else
 fi
 
 printf "${BLUE}%s${NORMAL}\n" "Updating Oh My Bash"
+OLDPWD="$PWD"
```

Comment:
`OLDPWD` is a special variable of Bash, and is automatically updated on `cd`. Thus we do not need this line.